library/pom.xml

```diff
@@ -325,6 +325,11 @@
       <version>7.0.6</version>
     </dependency>
     <dependency>
+      <groupId>com.yahoo.datasketches</groupId>
```

Comment:
@tweise The DataSketches library implements a number of streaming/sketch algorithms including hyperloglog and heavy hitters / frequent items and a "theta sketch" framework that will allow set operations on multiple streams, all of which I'm hoping to build operators for.